x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.test.tsx

```diff
@@ -186,45 +183,7 @@ describe.skip('Update Api Key', () => {
     fireEvent.click((await screen.findAllByTestId('selectActionButton'))[1]);
     expect(screen.getByTestId('collapsedActionPanel')).toBeInTheDocument();
 
-    fireEvent.click(await screen.findByText('Update API key'));
-    expect(screen.getByText('You will not be able to recover the old API key')).toBeInTheDocument();
-
-    fireEvent.click(await screen.findByText('Cancel'));
-    expect(
-      screen.queryByText('You will not be able to recover the old API key')
-    ).not.toBeInTheDocument();
-
-    fireEvent.click((await screen.findAllByTestId('selectActionButton'))[1]);
-    expect(screen.getByTestId('collapsedActionPanel')).toBeInTheDocument();
-
-    fireEvent.click(await screen.findByText('Update API key'));
-
-    fireEvent.click(await screen.findByTestId('confirmModalConfirmButton'));
-    await waitFor(() => expect(addSuccess).toHaveBeenCalledWith('Updated API key for 1 rule.'));
-    expect(bulkUpdateAPIKey).toHaveBeenCalledWith(expect.objectContaining({ ids: ['2'] }));
-    expect(screen.queryByText("You can't recover the old API key")).not.toBeInTheDocument();
-  });
-
-  it('Update API key fails', async () => {
-    bulkUpdateAPIKey.mockRejectedValueOnce(500);
-    renderWithProviders(<RulesList />);
-
-    expect(await screen.findByText('test rule ok')).toBeInTheDocument();
-
-    fireEvent.click((await screen.findAllByTestId('selectActionButton'))[1]);
-    expect(screen.getByTestId('collapsedActionPanel')).toBeInTheDocument();
-
-    fireEvent.click(await screen.findByText('Update API key'));
-    expect(screen.getByText('You will not be able to recover the old API key')).toBeInTheDocument();
-
-    fireEvent.click(await screen.findByText('Update'));
-    await waitFor(() =>
-      expect(addError).toHaveBeenCalledWith(500, { title: 'Failed to update the API key' })
```

Comment:
should we bring over this test as well?

Reply:
100%, I will make sure we go in the `catch` and call the toaster